### ISSUE TITLE
Update README.md with correct flags

### ISSUE THREAD

**Issue by Zamiell** [golangci/golangci-lint#340]
From the README:

```
go get -u github.com/golangci/golangci-lint
```

This results in the following error, which I presume is normal:

```
package github.com/golangci/golangci-lint: no Go files in /Users/james/go/src/github.com/golangci/golangci-lint
```

Is there a way to modify this command so that it does not throw an error? Perhaps with the -d flag? I'm unsure.

**Comment by AlekSi** [comment]
It should be
```
go get -u github.com/golangci/golangci-lint/cmd/golangci-lint
```

Please send a PR.

**Comment by Zamiell** [comment]
https://github.com/golangci/golangci-lint/pull/346
